specs/capri/Multi-attach.md

```diff
@@ -0,0 +1,136 @@
+# Title
+
+**Author(s)**: [Jack Li](https://github.com/jackhaibo)
+
+## Summary
+
+This design is proposed to enable multiAttach of volumes.
+
+## Motivation
+
+Currently we refuse to attach a volume that is already attached (in-use). There are situations where a cloud user may have deployed a clustered file-system on a Cinder volume and they would like the ability to attach a volume to multiple hosts/servers at the same time.
```

Comment:
s/Currently we refuse to attach a volume that is already attached (in-use)/Currently a request to attach a volume that is already attached (in-use) will be rejected

Remove "on a Cluster volume".  The multi-attach feature should not be limited to just Cinder

Reply:
Done.